src/applications/hca/tests/unit/utils/helpers/file-attachments.unit.spec.js

```diff
@@ -1,41 +1,34 @@
 import { expect } from 'chai';
 import { createPayload, parseResponse } from '../../../../utils/helpers';
 
-describe('hca `createPayload` method', () => {
-  let mockFile;
+describe('file-attachments', () => {
```

Comment:
I don't like this naming convention. All our tests start with `hca` and carry a much better description. Something like `hca file attachement methods` would be better here.

Reply:
Fixing it up now!